File: java/src/main/java/com/dokmatiq/docgen/model/ZugferdProfile.java
package com.dokmatiq.docgen.model;

import com.fasterxml.jackson.annotation.JsonValue;

/** ZUGFeRD profile level. */
public enum ZugferdProfile {
    MINIMUM("MINIMUM"), BASIC_WL("BASIC_WL"), BASIC("BASIC"),
    EN16931("EN16931"), EXTENDED("EXTENDED"), XRECHNUNG("XRECHNUNG");

    private final String value;

    ZugferdProfile(String value) { this.value = value; }

    @JsonValue
    public String getValue() { return value; }
}

File: php/src/Model/ZugferdProfile.php
<?php

declare(strict_types=1);

namespace Dokmatiq\DocGen\Model;

enum ZugferdProfile: string
{
    case MINIMUM = 'MINIMUM';
    case BASIC_WL = 'BASIC_WL';
    case BASIC = 'BASIC';
    case EN16931 = 'EN16931';
    case EXTENDED = 'EXTENDED';
    case XRECHNUNG = 'XRECHNUNG';
}
